Tools/VacuumTool/VacuumInput.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+using UnityEngine.InputNew;
+
+// GENERATED FILE - DO NOT EDIT MANUALLY
+namespace UnityEngine.InputNew
+{
+	public class VacuumInput : ActionMapInput {
+		public VacuumInput (ActionMap actionMap) : base (actionMap) { }
+		
+		public ButtonInputControl @secondary { get { return (ButtonInputControl)this[0]; } }
```

Comment:
Instead of naming this `secondary` it should reflect the action, which is to `vacuum`.